### PR TITLE
Bookstore CMS #1: Create a plain layout using React

### PR DESCRIPTION
**Pull request by danifromecuador**
## In this pull request I've:
- Created a `dev` branch and setup it as main branch in order to follow GitFlow pattern
- Created a `setup` branch
- Created a React App
- Added Microverse linters
- Created 2 pages: `Home` and `Categories` components
- Created `Navbar` component to link to the two routes corresponding to the 2 pages
- Inside `Home` page, I have created the following components:
  - `Book` that contains the individual book state, as well as a button to delete the book
  - `Books` this contains a list to renders individual books
  - `Input` this is a form to create new books
- Replaced the `category` input for `author` input
- The styling will be done in the next milestone
- Overview: 
image

### CODE REVIEW

File: src/components/Book.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          <div className="author">Suzanne Collins</div>
+        </div>
+        <div className="buttons">
+          <button type="button">Comments</button>
+          <hr />
+          <button type="button">Remove</button>
+          <hr />
+          <button type="button">Edit</button>
+        </div>
+      </div>
```
Comment (iambenkis) [inline src/components/Book.jsx]: - [ ] Good job creating the book component 👍. You repeated yourself , what if you have 100 books ? I suggest you to pass in props book's data such as `title` , `author` , etc to fix that issue. Remember that for good practices you should not repeat yourself .

```bash
const Book = (props) => {
  const { obj } = props;
  const { title, author } = obj;
  return (
    <div className="book-container">
    ....
   </>)
```

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @danifromecuador 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

- Use of react-router-dom package :heavy_check_mark:
- 2 `<Route>` added: :heavy_check_mark:
- `Books` components added :heavy_check_mark:

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/Book.jsx (hunk 1 above)
Comment (danifromecuador) [reply src/components/Book.jsx]: @iambenkis  Thank you dear reviewer for your review and also for the code example

**Review by sumairq — APPROVED** [review]
### Hi @danifromecuador , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**